exonum-java-binding-common/src/test/java/com/exonum/binding/common/proofs/list/ListProofHashCodeCalculatorTest.java

```diff
@@ -63,7 +64,8 @@ void visit_FullProof2elements() {
     calculator.visit(root);
 
     //calculate expected root hash
-    HashCode expectedRootHash = getBranchHashCode(getNodeHashCode(V1), getNodeHashCode(V2));
+    HashCode expectedRootHash = getBranchHashCode(getNodeHashCode(V1),
+        Optional.of(getNodeHashCode(V2)));
```

Comment:
It is not usually recommended to use Optional as an argument type. Here it reduces readability of the client code. A `@Nullable` argument or an overload would be better.

Reply:
Thanks, I'll update it.